packages/react/src/ReactElementValidator.js

```diff
@@ -44,6 +48,14 @@ if (__DEV__) {
       return element.type;
     } else if (element.type === REACT_FRAGMENT_TYPE) {
       return 'React.Fragment';
+    } else if (
+      typeof element.type === 'object' &&
+      element.type !== null &&
+      element.type.$$typeof === REACT_FORWARD_REF_TYPE
+    ) {
+      const functionName =
+        element.type.render.displayName || element.type.render.name || '';
+      return functionName !== '' ? `ForwardRef(${functionName})` : 'ForwardRef';
```

Comment:
Does this look like `getComponentName` duplication here? Can we unify?

Reply:
Yes, it's similar. But `getComponentName` accepts a `Fiber` and this is a `ReactElement`.

We could refactor both to use a shared helper, but it didn't seem like the obvious correct thing to do in this case.